docs/pages/blog/material-ui-is-now-mui.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Material-UI is now MUI
+description: Starting today, we are evolving our brand identity. We are clarifying the difference between our company and our products.
+date: 2021-09-15T00:00:00.000Z
+authors: ['oliviertassinari', 'danilo-leal', 'mbrookes']
+card: true
+---
+
+Starting today, we are evolving our brand identity. We are clarifying the difference between our company and our products.
+
+- Material-UI: the organization is now called [**MUI**](https://github.com/mui-org/).
+- Material-UI: the set of foundational MIT React components is now called [**MUI Core**](https://github.com/mui-org/material-ui).
+- Material-UI X: the set of advanced React components is now called [**MUI X**](https://github.com/mui-org/material-ui-x).
+
+The new identity matches with the increased scope of our focus.
```

Comment:
```suggestion
The new identity matches with the increased scope of our objectives.
```
?
("focus" seems at odds with "increased scope")

Reply:
I'm not keen on objectives because it sounds like we are going out to conquer. On the other hand "focus" resonates for me as caring.

Reply:
"increased scope" and "focus"seem opposed

Reply:
`increased scope of our vision`

Reply:
```suggestion
Our previous name was no longer serving our areas of focus.
We have grown our product offering.
We needed a new identity to match our increased scope.
```